src/private/functions.php

```diff
@@ -68,21 +68,8 @@ function _util_uri_apply_rules($uriPath, $incoming)
 
     // We can now be in one of 3 cases:
     //   1) At least one among DD_TRACE_RESOURCE_URI_FRAGMENT_REGEX and DD_TRACE_RESOURCE_URI_MAPPING_INCOMING|OUTGOING
-    //      is defined. Then ignore legacy DD_TRACE_RESOURCE_URI_MAPPING and apply *new normalization*.
-    //   2) Only DD_TRACE_RESOURCE_URI_MAPPING is defined, then apply *legacy normalization* for backward compatibility.
-    //   3) Nothing is defined, then apply *new normalization*.
-
-    // DEPRECATED: Applying legacy normalization for backward compatibility if preconditions are matched.
-    $legacyMappings = getenv('DD_TRACE_RESOURCE_URI_MAPPING');
```

Comment:
What was the motivation for removing this? Not necessary for this PR, but it might be a good idea to start adding checks for fully removed env vars to the startup logs/php info page.

Reply:
I can revert that if you wish - the motivation initially was some confusion about deprecation checks.

Reply:
Got it. I think we should probably keep this then unless there is a strong technical reason to remove it.

Reply:
Will revert that then tomorrow :-)